react/components/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+import {
+    FormControl,
+    FormHelperText,
+    FormControlLabel,
+    Radio,
+    RadioGroup,
+} from "@mui/material";
+import PropTypes from "prop-types";
+import InputLabel from "../InputLabel";
+
+const RadioButton = ({
+    id,
+    dataTestId,
+    options = [],
+    disabled = false,
+    label,
+    required = false,
+    error = false,
+    helperText,
+    ...rest
+}) => {
+    return (
+        <FormControl error={error} fullWidth>
+            <div style={{ width: 1, display: "flex", flexDirection: "column", flexGrow: 1}}/>
```

Comment:
do you think making flexDirection a prop will be helpful in the future? we can default it to "column"

Reply:
I copied the styling from other existing components and look like this is not even needed, It doesn't alter the layout of subcomponents.

Reply:
I understand what you meant, FlexDirection is going to create something like this, which is not so helpful for us.  
![image](https://user-images.githubusercontent.com/57495404/234883439-78ea0bc6-0e56-4760-b5e2-411922df0fee.png)

But We an still send a parameter "row" to this component and that parameter will be passed on to RadioGroup. For Material UI Radio Group "row" is accepted param and that is going to create something like this. 
![image](https://user-images.githubusercontent.com/57495404/234883878-cf87df2e-9b83-4243-a379-b2078a3bfff3.png)

Reply:
Gotcha. so top-level div was unnecessary and now we can pass the prop that will be taken care of by `...rest`.